### PR TITLE
Possibility to create youtube videos Media from mobile

### PR DESCRIPTION
**Pull request by clytemnestra**
The regex wasn't accepting mobile youtube videos.

### CODE REVIEW

**Comment by OskarStark** [comment]
@clytemnestra please add a test case and squash the commits afterwards!

thanky you for your work :+1:

**Comment by clytemnestra** [comment]
I'm not entirely sure where to get a valid fixture from. I believe I need to add one similar to [this one](https://github.com/sonata-project/SonataMediaBundle/blob/master/Tests/fixtures/valid_youtube.txt) but can't figure out where to get it from. Or is it hand-written?

File: Tests/Provider/YouTubeProviderTest.php [hunk 1]
```diff
@@ -157,6 +157,10 @@ public static function getUrls()
             array('http://www.youtube.com/watch?v=BDYAbAtaDzA#t=0m10s'),
             array('http://www.youtube.com/embed/BDYAbAtaDzA?rel=0'),
             array('http://www.youtube.com/watch?v=BDYAbAtaDzA'),
+			array('http://www.m.youtube.com/watch?v=BDYAbAtaDzA'),
```
Comment (OskarStark) [inline Tests/Provider/YouTubeProviderTest.php]: please fix the indentation

**Comment by OskarStark** [comment]
and please squash your commits

**Comment by clytemnestra** [comment]
Like so? Sorry it's like the first time I do this kind of stuff.

**Comment by OskarStark** [comment]
commit looks good, but the indention is wrong again :smile:

**Comment by OskarStark** [comment]
execute `make cs`

**Comment by clytemnestra** [comment]
I'm confused. Here it looks like it's badly indented, but [here](https://github.com/clytemnestra/SonataMediaBundle/blob/patch-1/Tests/Provider/YouTubeProviderTest.php#L153) it looks fine. It also looks fine on my pc.

**Comment by core23** [comment]
Go to the root directory of your local copy and run the terminal with ``make cs``

**Comment by clytemnestra** [comment]
`george@george-VirtualBox:~/Documents/asd/SonataMediaBundle$ make cs
php-cs-fixer fix --verbose
Loaded config from "/home/george/Documents/asd/SonataMediaBundle/.php_cs"
............................................................................................................................................................................................................................................
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error
Fixed all files in 25.911 seconds, 9.500 MB memory used
`

No changes made to commit.

**Comment by core23** [comment]
Can you verfify that you got the latest version of the php-cs-fixer (``php-cs-fixer --version``)

**Comment by clytemnestra** [comment]
PHP CS Fixer version 1.11.1 by Fabien Potencier (2c9f829)

george@george-VirtualBox:~/Documents/asd/SonataMediaBundle$ sudo php-cs-fixer self-update
php-cs-fixer is already up to date.

**Comment by OskarStark** [comment]
strange, can you adjust it manually?

**Comment by clytemnestra** [comment]
I believe GitHub is acting weird. This is the file without any modifications, freshly cloned project and nothing to commit.   http://i.imgur.com/QaLZEwA.png?1

**Comment by clytemnestra** [comment]
I think GitHub is acting weird. This is the file from a fresh clone and there is nothing to commit.

**Comment by OskarStark** [comment]
maybe invisible characters?

**Comment by clytemnestra** [comment]
Did that fix it?

**Comment by OskarStark** [comment]
it does, thank you for your work, what did you do?

**Comment by clytemnestra** [comment]
Phew. 

I deleted all spaces and tabs and reindented it manually.

**Comment by OskarStark** [comment]
that was what i thoght

thank you!

**Pull request merged**